pasos-para-investigar.md

```diff
@@ -0,0 +1,51 @@
+## Pasos para Investigar
+
+1. **Recopilar información (datos científicos, libros u otras fuentes confiables)**
+2. **Entender la taxonomía de la planta**
+	- Dominio
+	- Reino
+	- Filo o división
+	- Clase
+	- Orden
+	- Familia
+	- Género
+	- Especie
+
+Es importante entender la taxonomía para realizar la investigación con un enfoque claro de lo que estamos buscando.
```

Comment:
This sentence should come before the list. It introduces the section. 

How does understanding the taxonomy help you conduct research about the plant?